api/v1/database_types.go

```diff
@@ -220,6 +224,39 @@ type ExtensionSpec struct {
 	Schema string `json:"schema,omitempty"`
 }
 
+// FDWSpec configures an Foreign Data Wrapper in a database
+type FDWSpec struct {
+	DatabaseObjectSpec `json:",inline"`
+
+	// Name of the handler function (e.g., "postgres_fdw_handler")
+	Handler string `json:"handler,omitempty"`
+
+	// Name of the validator function (e.g., "postgres_fdw_validator")
+	Validator string `json:"validator,omitempty"`
+
+	// Owner specifies the database user who will own the Foreign Data Wrapper.
+	// By default, the owner of a new FDW is the current session user.
+	// Even if an owner is explicitly specified during creation, it will be ignored.
+	Owner string `json:"owner,omitempty"`
```

Comment:
```suggestion
	// Owner specifies the database user who will own the Foreign Data Wrapper.
	// By default, the owner of a new FDW is the current session user.
	// Even if an owner is explicitly specified during creation, it will be ignored.
	// +optional
	Owner string `json:"owner,omitempty"`
```

Any field that has the JSON attribute `omitempty` needs to be marked as `+optional`, otherwise when we generate the documentation it is marked as required.

Reply:
Additionally, given that the owner can only be a superuser and I don't really see why anyone should change it from the default, I wonder if we should just drop this field.